src/mobile/src/ui/routes/navigation.js

```diff
@@ -26,52 +30,66 @@ import ForceChangePassword from 'ui/views/wallet/ForceChangePassword';
 import SeedVaultBackupComponent from 'ui/views/onboarding/SeedVaultBackup';
 import { isIPhoneX } from 'libs/device';
 
-function getGenerator(screen) {
+function applyHOCs(screen) {
     if (isIPhoneX) {
-        return withSafeAreaView(screen);
+        screen = withSafeAreaView(screen);
```

Comment:
Can't think the affect reassignment can cause here, but I would suggest to avoid it.